extensions/yaml_support/_parser/base_parser.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABCMeta, abstractmethod
+from typing import TypeVar, List
+
+T = TypeVar("T")
```

Comment:
from typing import Any can be used instead.

Reply:
Done